contracts/NFTStorefront.cdc

```diff
@@ -448,6 +551,38 @@ pub contract NFTStorefront {
             return self.listings.keys
         }
 
+        // getDublicateListingIDs
+        // Returns an array of listing Ids that are duplicate for the given `nftType` and `nftId`.
+        //
+        pub fun getDuplicateListingIDs(nftType: Type, nftID: UInt64, listingID: UInt64): [UInt64] {
+            let listingIds = self.listedNFTs[nftType][nftID]
+            // Verify that given listing Id also a part of the `listingIds`
+            let isListingExisted = listingIds.contains(listingID)
```

Comment:
Can you fix the grammar here? I think it should be something like `doesListingExist`